indra/newview/llavataractions.cpp

```diff
@@ -285,25 +285,24 @@ void LLAvatarActions::startAdhocCall(const uuid_vec_t& ids, const LLUUID& floate
     make_ui_sound("UISndStartIM");
 }
 
-/* AD *TODO: Is this function needed any more?
-    I fixed it a bit(added check for canCall), but it appears that it is not used
-    anywhere. Maybe it should be removed?
 // static
-bool LLAvatarActions::isCalling(const LLUUID &id)
+bool LLAvatarActions::canCall()
 {
-    if (id.isNull() || !canCall())
+    if (LLVoiceClient* voice_client = LLVoiceClient::getInstance())
```

Comment:
voice_client is a singleton, there shouldn't be a need to check it.

Reply:
The null-check is removed